### PR TITLE
Use absolute target names in rapidcheck

### PR DESCRIPTION
**Pull request by tapia**
Conan feature related: https://github.com/conan-io/conan/pull/10099

### CODE REVIEW

**Comment by SpaceIm** [comment]
Other components also want non-namespaced targets in set_property()

File: recipes/rapidcheck/all/conanfile.py [hunk 1]
```diff
@@ -123,6 +123,7 @@ def _module_file_rel_path(self):
                             "conan-official-{}-targets.cmake".format(self.name))
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "rapidcheck")
```
Comment (czoido) [inline recipes/rapidcheck/all/conanfile.py:126]:
```suggestion
```
This is not needed

Comment (SpaceIm) [reply recipes/rapidcheck/all/conanfile.py:126]: nop, but it's explicit, at least for me it's easier to maintain recipes explicitly declaring official config files.

**Review by czoido — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
LGTM

**Review by lasote — APPROVED** [review]

**Pull request merged**